types.go

```diff
@@ -74,7 +78,9 @@ type Field struct {
 	MapKeyType    string `json:"mapKeyType,omitempty" yaml:"mapKeyType,omitempty"`
 	MapValueType  string `json:"mapValueType,omitempty" yaml:"mapValueType,omitempty"`
 
-	StructFields map[string]Field `json:"structFields,omitempty" yaml:"structFields,omitempty"`
+	StructFields              map[string]Field              `json:"structFields,omitempty" yaml:"structFields,omitempty"`
```

Comment:
I'm personally not the biggest fan of having separate structs to reference, I'd rather put the validation/binding tags on the field to be referenced there - unless that's what you're already doing an I'm not reading correctly

Reply:
Nevermind I did misread